### PR TITLE
Rework mc admin prometheus reference page

### PR DESCRIPTION
**Pull request by feorlen**
Split the `mc admin prometheus` page into subpages and add missing docs for `mc admin prometheus metrics`.

Staged
http://192.241.195.202:9000/staging/DOCS-1255/linux/reference/minio-mc-admin/mc-admin-prometheus.html

Fixes https://github.com/minio/docs/issues/1255

### CODE REVIEW

File: source/reference/minio-mc-admin/mc-admin-policy-attach.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ Exactly one :mc-cmd:`~mc admin policy attach --user` or one :mc-cmd:`~mc admin p
 
    .. tab-item:: EXAMPLE
 
-      The following command displays the current in-progress S3 API calls on the :term:`alias` ``myminio``.
+      The following command attaches the ``readonly`` policy to the user ``james`` on the deployment at :term:`alias` ``myminio``.
```
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-policy-attach.rst:33]: drive-by fix elsewhere

File: source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+      Valid values are:
+
+      - ``bucket``
+      - ``cluster``
+      - ``node``
+      - ``resource``
+
+      If not specified, the command returns cluster metrics.
+      Cluster metrics also include node metrics.
+      The output includes additional information about each metric, such as if its value is a ``counter`` or ``gauge``.
```
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst:84]: a prom-knowledgeable person would already know this, but it helps others understand what to expect.

File: source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+Use :mc-cmd:`mc admin prometheus metrics` to print bucket metrics for a MinIO deployment:
+
+.. code-block:: shell
+   :class: copyable
+
+      mc admin prometheus metrics ALIAS bucket
+
+- Replace ``ALIAS`` with the :mc-cmd:`alias <mc alias>` of the MinIO deployment.
+
+The output resembles the following:
```
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst:109]: This is the output from a toy deployment with a single bucket. It's still huge. It's at the bottom of the page at least?

Could truncate.

File: source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+
+      Valid values are:
+
+      - ``bucket``
+      - ``cluster``
+      - ``node``
+      - ``resource``
+
+      If not specified, the command returns cluster metrics.
+      Cluster metrics also include node metrics.
```
Comment (ravindk89) [inline source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst:83]:
```suggestion
      Cluster metrics included rollups of certain node metrics.
```

File: source/reference/minio-mc-admin/mc-admin-prometheus.rst [hunk 5]
```diff
@@ -15,57 +15,36 @@ Description
 
 .. start-mc-admin-prometheus-desc
 
-The :mc-cmd:`mc admin prometheus` command generates a configuration file for
-use with `Prometheus <https://prometheus.io/>`__.
+The :mc:`mc admin prometheus` command and its subcommands provide access to MinIO Prometheus metrics.
```
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-prometheus.rst:18]: This description could be better, nothing's coming to me.

File: source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst (hunk 4 above)
Comment (ravindk89) [reply source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst:83]: Spitballing - it's not all node metrics, only some, and I don't think we have a good difflist.

**Review by ravindk89 — APPROVED** [review]
No notes. LGTM

File: source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst (hunk 4 above)
Comment (feorlen) [reply source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst:83]: Yeah I was wondering what to do about that.

Comment (feorlen) [reply source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst:83]: I'm gonna go with your edit then we can figure out what else to say. I also am still unclear which node metrics or if that will change.

File: source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+
+      Valid values are:
+
+      - ``bucket``
+      - ``cluster``
+      - ``node``
+      - ``resource``
+
+      If not specified, the command returns cluster metrics.
+      Cluster metrics included rollups of certain node metrics.
```
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst:83]: typo
```suggestion
      Cluster metrics include rollups of certain node metrics.
```
